decidim-core/app/models/decidim/action_log.rb

```diff
@@ -194,7 +194,7 @@ def user_lazy(cache: true)
     # as a regular object.
     def participatory_space_lazy(cache: true)
       return if participatory_space_id.blank? || participatory_space_type.blank?
-      return resouce_lazy if participatory_space_id == resource_id && participatory_space_type == resource_type
+      return resource_lazy if participatory_space_id == resource_id && participatory_space_type == resource_type
```

Comment:
notable

Reply:
Great finding ! I search through the project and i could not find anything named like "res**ouc**e_lazy"